indico/modules/events/layout/__init__.py

```diff
@@ -45,7 +45,8 @@
     'timetable_theme': None,
     'use_custom_menu': False,
     'timetable_by_room': False,
-    'timetable_detailed': False
+    'timetable_detailed': False,
+    'weeks_view_settings': None
```

Comment:
too specific, should be `theme_settings: {}` and cleared whenever the theme is changed (see my other comments below regarding how this should be much more generic)